build.sh

```diff
@@ -52,3 +52,20 @@ if [ "$RUN_TESTS" -ne 0 ]; then
 	$RUNTIME_CMD run --entrypoint cat "${REPO}${DRIVER}" \
 		/profile.cov >profile.cov
 fi
+
+DRIVER="glustervirtblock-csi-driver"
```

Comment:
The driver name looks bit confusing to me, if we meant this is a virtual block driver we could work on better names

Reply:
As discussed in gcs channel, keeping it as glustervirtblock.